webapp/components/notifications/NotificationMenu/NotificationMenu.spec.js

```diff
@@ -46,11 +46,46 @@ describe('NotificationMenu.vue', () => {
       expect(wrapper.contains('.dropdown')).toBe(false)
     })
 
+    describe('given only unread notifications', () => {
+      beforeEach(() => {
+        data = () => {
+          return {
+            displayedNotifications: [
+              {
+                id: 'notification-41',
+                read: true,
+                post: {
+                  id: 'post-1',
+                  title: 'some post title',
+                  contentExcerpt: 'this is a post content',
+                  author: {
+                    id: 'john-1',
+                    slug: 'john-doe',
+                    name: 'John Doe',
+                  },
+                },
+              },
+            ],
+          }
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-23T10:54:19Z" title="Monday, September 23rd 2019, 12:54:19 pm +02:00">Sep 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -79,12 +79,26 @@ describe('NotificationMenu.vue', () => {
                   },
                 },
               },
+              {
+                id: 'notification-43',
+                read: true,
+                post: {
+                  id: 'post-3',
+                  title: 'red post title',
+                  contentExcerpt: 'this is yet another post content',
+                  author: {
+                    id: 'john-1',
+                    slug: 'john-doe',
+                    name: 'John Doe',
+                  },
+                },
+              },
             ],
           }
         }
       })
 
-      it('displays the total number of notifications', () => {
+      it('displays the unred number of notifications', () => {
```


```suggestion
      it('displays the unread number of notifications', () => {
```
Or do you mean unred, e.g. blue? :wink:

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-23T11:43:56Z" title="Monday, September 23rd 2019, 1:43:56 pm +02:00">Sep 23, 2019</time>_

---


Ups I wrote the past tense after its phonetic that is like „red“. Sorry!